rust/rope/src/rope.rs

```diff
@@ -708,6 +708,16 @@ impl fmt::Debug for Rope {
     }
 }
 
+impl fmt::Display for Rope {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        if f.alternate() {
```

Comment:
Thinking about this, I think that the _default_ representation should just print the contents of the rope, similar to how the default representation of `String` just prints the contents of the String. In general, we would like to behave more or less the same as the built in string types do.